src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.ObjectCreationPlaceholderLocal.cs

```diff
@@ -8,7 +8,7 @@
 
 namespace Microsoft.CodeAnalysis.CSharp
 {
-    internal partial class DataFlowPass
+    internal partial class NullableWalker
```

Comment:
>NullableWalker [](start = 27, length = 14)

Possibly want to change the file name too.

Reply:
Renamed file.

---
In reply to: [148388390](https://github.com/dotnet/roslyn/pull/22932#discussion_r148388390) [](ancestors = 148388390)